### DISCUSSION TITLE
Suggestion for 2 appllications to be included in the default software list in Linux Mint

### DISCUSSION THREAD

**Discussion by valeriopincini** [linuxmint/discussions#544] [Ideas]
Hi everyone
I would like to suggest 2 more apps to be included in the default software list:
- Bleachbit
- Audacious
I would like also that Audacious will become the default application to play audio files.
What do you think about it?

**Comment by Jopp-gh** [comment]
Check this link for "BleachBit":
https://easylinuxtipsproject.blogspot.com/p/fatal-mistakes.html?m=1

**Reply by valeriopincini** [reply]
Hi Jopp-gh
I have checked out the link you posted.
I am sorry but I do not agree with your opinion on Bleachbit and I tell you why.
The article says "In short: it's a newbie trap".
I use Linux at home since 1992 and at work since 2018; I am definetively not a newbie to Linux :-)
My opinion on Bleachbit comes from direct experience: I have been using it for 10 years on Debian based systems (mainly Debian, Lmde, Linux Mint) without having a single problem.
You have complete control on what you purge and, if you know your way, it is very useful to avoid "bloating" your system.
For example I think it does a wonderful job in purging all installed applications' locales for languages I  do not use.
Experience apart Bleachbit has been around since 2008; if it was a newbie trap it would not have "lived" for so long.
I don't want to convince you about but I want to clarify that I am not suggesting this application lightly.
That said thank you for finding the time of commenting my post.
Cheers

**Reply by Jopp-gh** [reply]
>The article says "In short: it's a newbie trap".

Well, this assertion is generally true. But I shared that article not because of that one sentence. The linked guide is carefully written and  the points definitively helpful. 

I've used similar cleaning tools on Mac and windows as well and prefer to stick to good "defaults" for system relevant data at least.
Even with computers, a lot can go wrong if you modify (I'm not even talking about deleting data) the wrong files or strings.

I'm sorry if your felt bad because of some details in that guide.
